README.md

```diff
@@ -1 +1,94 @@
-# container-resources-policy
+## Container resources policy
+
+This policy is designed to enforce constraints on the resource requirements  of
+Kubernetes containers. It follows a two-step verification process: initially
+checking whether the container has defined resource requirements, and
+subsequently ensuring that these resources fall within the permissible range
+set by the maximum resource requirements configured in the policy settings.
+
+## Configuration
+
+Users can configure the policy using the following parameters in YAML format:
+
+```yaml
+# optional
+memory:
+  defaultRequest: "100M"
+  defaultLimit: "500M"
+  maxLimit: "4G"
+# optional
+cpu:
+  defaultRequest: 100m
+  defaultLimit: 200m
+  maxLimit: 500m
+# optional
+ignoreImages: ["ghcr.io/foo/bar:1.23", "myimage", "otherimages:v1"]
+```
+
+Users can skip the optional parts of the configuration, but an empty configuration is not
+allowed. Thus, at least one of the configurations,  `cpu`, or `memory` should
+be defined including all their sub fields.  All CPU and memory configuration
+should be expressed using the [quantity
+definitions](https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/)
+of Kubernetes.
+
+The `ignoreImages` values does not need to define the image tag. If the container
+image tag is omitted, any tag of the given image will be skipped. If the tag is
+provided, only that tag will be skipped in the policy evaluation. 
+The policy verifies the consistency of the values provides:
```

Comment:
No, this is using `*`. Please, just copy the snippet about this parameter from the original Rego policy

Reply:
Oh... I forgot to update this. Sorry.